src/redux/categories/categories.js

```diff
@@ -0,0 +1,23 @@
+const COMPLETED = 'COMPLETED';
+const INPROGRESS = 'INPROGRESS';
```

Comment:
- [ ] You need to import the value of action type

Reply:
hi @GzimAsani, am a little confused 😕 , if you please can you help me understand from where should I import the value. 🔢 
thanks in advance bro.